src/isa/riscv64/instr/rvf/decode.h

```diff
@@ -67,38 +67,46 @@ static inline def_DHelper(r2fr){
 #ifdef CONFIG_RVV
 bool vp_enable();
 def_THelper(vload) {
-  def_INSTR_TAB("??? ?00 ? ????? ????? ??? ????? ????? ??", vlduu);
-  def_INSTR_TAB("??? ?01 ? ????? ????? ??? ????? ????? ??", vldxu);
-  def_INSTR_TAB("??? ?10 ? ????? ????? ??? ????? ????? ??", vldsu);
-  def_INSTR_TAB("??? ?11 ? ????? ????? ??? ????? ????? ??", vldxu);
+  def_INSTR_TAB("??? 000 ? ?0000 ????? ??? ????? ????? ??", vle);
+  def_INSTR_TAB("??? 000 ? 01000 ????? ??? ????? ????? ??", vlr);
+  def_INSTR_TAB("??? 000 ? 01011 ????? ??? ????? ????? ??", vlm);
+  def_INSTR_TAB("??? 001 ? ????? ????? ??? ????? ????? ??", vlxe);
+  def_INSTR_TAB("??? 010 ? ????? ????? ??? ????? ????? ??", vlse);
+  def_INSTR_TAB("??? 011 ? ????? ????? ??? ????? ????? ??", vlxe);
   return EXEC_ID_inv;
 }
 
 def_THelper(vstore) {
   decode_op_i(s, id_src2, (sword_t)s->isa.instr.i.simm11_0, false);
   decode_op_fr(s, id_dest, s->isa.instr.i.rd, false);
-  def_INSTR_TAB("??? ?00 ? ????? ????? ??? ????? ????? ??", vstu);
-  def_INSTR_TAB("??? ?01 ? ????? ????? ??? ????? ????? ??", vstx);
-  def_INSTR_TAB("??? ?10 ? ????? ????? ??? ????? ????? ??", vsts);
-  def_INSTR_TAB("??? ?11 ? ????? ????? ??? ????? ????? ??", vstx);
+  def_INSTR_TAB("??? 000 ? 00000 ????? ??? ????? ????? ??", vse);
+  def_INSTR_TAB("??? 000 ? 01000 ????? ??? ????? ????? ??", vsr);
+  def_INSTR_TAB("??? 000 ? 01011 ????? ??? ????? ????? ??", vsm);
+  def_INSTR_TAB("??? 001 ? ????? ????? ??? ????? ????? ??", vsxe);
+  def_INSTR_TAB("??? 010 ? ????? ????? ??? ????? ????? ??", vsse);
+  def_INSTR_TAB("??? 011 ? ????? ????? ??? ????? ????? ??", vsxe);
```

Comment:
PLZ add some comments at the end of each line, like VSE, VSUXE, VSSE, VSOXEI, VSM